### DISCUSSION TITLE
I Want to Get Wind Data from Simulation.

### DISCUSSION THREAD

**Discussion by ubeydullahsu** [microsoft/AirSim#3651] [Support Q&A]
Hi everyone. I want to get the wind data from simulation. I have tried to write simGetWind() function. But I am getting this error:

msgpackrpc.error.RPCError: rpclib: server could not find function 'simGetWind' with argument count 0.

How can I solve this problem or is there any alternative way to get wind data?

**Comment by rajat2004** [comment]
So you're trying to add a new API `simGetWind()`? You might have missed adding the RPC wrapper, see https://github.com/microsoft/AirSim/pull/3611 and https://github.com/microsoft/AirSim/pull/2867 which added the Wind implementation. Any suggestions on the doc PR are very much welcome!

Aside from this, I personally don't see much point in this API itself. The wind values can be set in the settings and the API, so you already know what value is set currently. Just my opinion though
